tools/internal_ci/linux/grpc_xds_k8s_xlang.sh

```diff
@@ -44,13 +49,14 @@ readonly VERSION_TAG="master v1.50.x"
 run_test() {
   # Test driver usage:
   # https://github.com/grpc/grpc/tree/master/tools/run_tests/xds_k8s_test_driver#basic-usage
-  local tag="${1:?Usage: run_test tag server_lang client_lang}"
-  local slang="${2:?Usage: run_test tag server_lang client_lang}"
-  local clang="${3:?Usage: run_test tag server_lang client_lang}"
-  local server_image_name="${IMAGE_REPO}/${slang}-server:${tag}"
-  local client_image_name="${IMAGE_REPO}/${clang}-client:${tag}"
+  local stag="${1:?Usage: run_test server_tag client_tag server_lang client_lang}"
+  local ctag="${2:?Usage: run_test server_tag client_tag server_lang client_lang}"
+  local slang="${3:?Usage: run_test server_tag client_tag server_lang client_lang}"
+  local clang="${4:?Usage: run_test server_tag client_tag server_lang client_lang}"
+  local server_image_name="${IMAGE_REPO}/${slang}-server:${stag}"
+  local client_image_name="${IMAGE_REPO}/${clang}-client:${ctag}"
   # TODO(sanjaypujare): skip test if image not found (by using gcloud_gcr_list_image_tags)
-  local out_dir="${TEST_XML_OUTPUT_DIR}/${tag}/${clang}-${slang}"
+  local out_dir="${TEST_XML_OUTPUT_DIR}/${ctag}-${stag}/${clang}-${slang}"
```

Comment:
We might want to reverse it now, and have `${clang}-${slang}` in front. It really depends how we want to break it down in report dashboards.

Reply:
As discussed reversing is not required because of the structure of our dashboard.